### PR TITLE
Domain classes

### PR DESCRIPTION
**Pull request by LalithaNadimpalli**
Basic domain

### CODE REVIEW

**Review by sushmakoosukuntla — COMMENTED** [review]
why do we need duration, when we have start and end time?

**Comment by LalithaNadimpalli** [comment]
Thought it will be good to provide how long the event going to be..when you provide the start and end time we have to calculate in mind how long the event going to last right so just thought we shouldn't give that difficulty to user..

File: EventCatalogAPI/Domain/CatalogEvent.cs [hunk 1]
```diff
@@ -10,12 +10,17 @@ public class CatalogEvent
...
+        public decimal Price{ get; set; }
         public string EventImageUrl { get; set; }
-        public string EventStartTime { get; set; }
-        public string EventEndTime { get; set; }
+        public float EventStartTime { get; set; }
+        public float EventEndTime { get; set; }
+        public int Duration { get; set; }
+        public int EventTypeId { get; set; }
+        public int EventCategoryId { get; set; }
+        public CatalogCategory CatalogCategory { get; set; }
```
Comment (misurac) [inline EventCatalogAPI/Domain/CatalogEvent.cs:22]: Why do we have line 22 and 23?

Comment (LalithaNadimpalli) [reply EventCatalogAPI/Domain/CatalogEvent.cs:22]: I am trying to establish a relationship between all the models..In future the Event class can talk to Type and Category classes with the help of the foreign key(EventTypeId and EventCategoryId)

Comment (misurac) [reply EventCatalogAPI/Domain/CatalogEvent.cs:22]: I guess I'm mostly confused about the naming. What do you think about renaming the class as EventCategory? Trying to understand why it is called Catalog Category

Comment (LalithaNadimpalli) [reply EventCatalogAPI/Domain/CatalogEvent.cs:22]: sure we could do that

**Pull request closed without merging**